tests/test_subscriptions.py

```diff
@@ -158,6 +159,8 @@ def test_get(self):
             requests.codes.okay,
             headers=get_auth_header())
         json_response = resp_obj.json
+        self.addCleanup(self._cleanup_subscription, json_response['uuid'])
```

Comment:
this should go before your first `assert` and preferable as soon as you create the subscription.